### PR TITLE
[DOC-640] Migrate Marketplace EQP guide

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) migrates the content of the Marketplace EQP API Guide to the `commerce-marketplace` repo, per https://jira.corp.magento.com/browse/DOC-640

## Affected pages

-  All

## Links to related PRs or Jira tickets

-  ...

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My changes follow the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/eqp/v1/access-keys.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+-  From the list of your API access keys, you can **delete** the key.
+   -  Once deleted, the key can no longer be used to obtain a [session token](auth.md#how-to-use-a-session-token).
+   -  However, any **session tokens** obtained from the previous key will remain valid, until those session tokens expire.
+
+<InlineAlert variant="info" slots="text"/>
+
+Marketplace EQP API **access keys** generated in the [sandbox][2] environment are separate from those generated in [production][1].
+You cannot use access keys from one environment in the other.
+<br/><br/>
+**Session tokens** generated in the **sandbox** are separate from those generated in **production**.
```
Comment (jeff-matthews) [inline src/pages/guides/eqp/v1/access-keys.md:61]: This isn't rendering the way I think you want it to.

Screen Shot 2022-04-01 at 2 10 48 PM

File: src/pages/guides/eqp/v1/packages.md [hunk 2]
```diff
@@ -0,0 +1,1056 @@
...
+*  Additional details are provided in the two main EQP areas:
+   *  **technical** - Provides the current technical status.
+   *  **marketing** - Provides the current marketing status.
+
+##### Overall Status
+
+|Value|Type|Description|
+|-----|----|-----------|
+|`draft`|string|The submission is in the draft state, not yet submitted to the EQP process.|
+|in_progress|string|The submission is in the EQP pipeline. Refer to the `technical` and `marketing` status for more details.|
```
Comment (jeff-matthews) [inline src/pages/guides/eqp/v1/packages.md:530]:
```suggestion
|`in_progress`|string|The submission is in the EQP pipeline. Refer to the `technical` and `marketing` status for more details.|
```

File: src/pages/guides/eqp/v1/packages.md [hunk 3]
```diff
@@ -0,0 +1,1056 @@
...
+
+You can update one or more parameters in `draft` mode.
+In this mode, the API checks only for basic type-validation issues.
+
+When the `action` field is set to `submit`, the API validates fields to ensure all
+required parameters are available on the Developer Portal to initiate the EQP process,
+and that parameters which depend upon each other match up correctly.
+
+<InlineAlert variant="info" slots="text"/>
+All `action` fields are optional. If not specified, `draft` is the default value.
```
Comment (jeff-matthews) [inline src/pages/guides/eqp/v1/packages.md:587]: The backticks aren't rendering code format...

Screen Shot 2022-04-01 at 2 15 23 PM

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
I'm not sure what to do with the [packages table ](https://developer-stage.adobe.com/commerce/marketplace/guides/eqp/v1/packages/#package-fields).

The only options I can think of are to:

- Convert to HTML and try setting narrow column widths
- Move the table to a separate page that uses a full-width layout and link to it
- Ask the [adobe/aio-theme](https://github.com/adobe/aio-theme) maintainers what they recommend

**Comment by keharper** [comment]
I went with option 4: blow it all up. 

The entries in the first column of the table are now individual h3 or h4s, while the last two columns were transformed into descriptions. The middle 4 columns still exist as a lot of individual 1-row tables.

The descriptions for some of the more complex fields in the original table linked to sections that provided code samples and/or more detailed explanations. These have been merged into the new h3/h4 sections.

Before:
Screen Shot 2022-04-04 at 4 58 38 PM

After:
Screen Shot 2022-04-04 at 4 57 50 PM





Review: 
https://developer-stage.adobe.com/commerce/marketplace/guides/eqp/v1/packages/

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**